client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js

```diff
@@ -0,0 +1,263 @@
+			header: translate( 'Find the perfect premium WordPress theme for your blog or website.' ),
+			description: translate(
+				"Browse all premium themes for WordPress.com. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
+			),
+		},
+		marketplace: {
+			title: translate( 'Paid WordPress Themes' ),
+			header: translate( 'Find the perfect paid WordPress theme for your blog or website.' ),
+			description: translate(
+				"Browse all paid themes for WordPress.com. Activate the one that's right for you, risk-free for 14 days."
+			),
+		},
+	},
+	blog: {
+		all: {
+			title: translate( 'Blog WordPress Themes' ),
+			header: translate( 'Find the perfect theme for your blog.' ),
+			description: translate(
+				"Start your blog with the perfect theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		free: {
+			title: translate( 'Free Blog WordPress Themes' ),
+			header: translate( 'Discover the perfect free theme for your blog.' ),
+			description: translate(
+				"Launch your blog with the perfect free theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		premium: {
+			title: translate( 'Premium Blog WordPress Themes' ),
+			header: translate( 'Choose premium themes for your blog.' ),
+			description: translate(
+				"Enhance your blog with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
+			),
+		},
+		marketplace: {
+			title: translate( 'Paid Blog WordPress Themes' ),
+			header: translate( 'Select paid themes for your blog.' ),
+			description: translate(
+				"Improve your blog with the perfect paid theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you."
+			),
+		},
+	},
+	business: {
+		all: {
+			title: translate( 'Business WordPress Themes' ),
+			header: translate( 'Find the perfect theme for your business website.' ),
+			description: translate(
+				"Start your business website with the perfect theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		free: {
+			title: translate( 'Free Business WordPress Themes' ),
+			header: translate( 'Discover the perfect free theme for your business website.' ),
+			description: translate(
+				"Launch your business website with the perfect free theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		premium: {
+			title: translate( 'Premium Business WordPress Themes' ),
+			header: translate( 'Choose premium themes for your business website.' ),
+			description: translate(
+				"Enhance your business website with the premium paid theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
```

Comment:
This indeed matches the spreadsheet, but I think the spreadsheet wrong and this should be like this:

```suggestion
				"Enhance your business website with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
```